app/views/snps/_snpedia_papers.html.erb

```diff
@@ -0,0 +1,27 @@
+<div id="snpedia">
+  <h3>Links to SNPedia</h3>
+  <% if snp.snpedia_papers.present? %>
+    <table class="table table-striped">
+      <tr>
+        <th>Title</th>
+        <th>Summary</th>
+      </tr>
+      <% snp.snpedia_papers.each do |p| %>
+        <tr>
+          <td>
+            <%= bold_if_matching_local_genotype(p, user_snp) do %>
+              <%= link_to "#{snp.name} #{p.local_genotype.split('').join('/')}", p.url %>
+            <% end %>
+          </td>
+          <td><b><%= p.summary.capitalize %></b></td>
+        </tr>
+      <% end %>
+    </table>
+  <% else %>
+    There are no SNPedia-links yet. Please come back later.
+  <% end %>
+  <div class="small">
+    Note: For some SNPs, testing providers detect the genotype from the opposite strand of DNA, so that the genotypes listed at openSNP don't appear here. In those cases, please replace "A" by "T" or "G" by "C" (and vice-versa).
```

Comment:
Yes, this means the users have to do the substitution in their head. You could try to match for the opposite strand in some instances. But there's the problem where the variants are C & G or A and T and in those cases it's impossible to decide whether to highlight the flipped strand or the given one. Does that make sense?